src/extension.js

```diff
@@ -146,32 +124,12 @@ function activate(context) {
     "darkSynthwave84.disableNeon",
     uninstall
   );
-  let whatsNew = vscode.commands.registerCommand(
-    "darkSynthwave84.whatsNew",
-    showUpdatePage
-  );
 
 	context.subscriptions.push(disposable);
 	context.subscriptions.push(disable);
-	context.subscriptions.push(whatsNew);
 }
 exports.activate = activate;
 
-
-function showUpdatePage() {
```

Comment:
What if this feature is needed though? Is there no other to get arrount this bug?